pylint/test/test_functional.py

```diff
@@ -226,6 +230,7 @@ def __init__(self, test_file):
         self._linter.set_reporter(test_reporter)
         self._linter.config.persistent = 0
         checkers.initialize(self._linter)
+        register_check_consider_merging_isinstance(self._linter)
```

Comment:
You don't have to do this here. Even if we are moving the checker into the core, you should not change test_functional in this way for registering additional checkers.